build/CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ option(TRTIS_ENABLE_CUSTOM "Include Custom backend in server" OFF)
 option(TRTIS_ENABLE_HTTP "Include HTTP API in server" ON)
 option(TRTIS_ENABLE_GRPC "Include GRPC API in server" ON)
 option(TRTIS_ENABLE_HTTP_V2 "Enable version 2 of the HTTP API in server" OFF)
-option(TRTIS_ENABLE_GRPC_V2 "Enable version 2 of the GRPC API in server" OFF)
+option(TRTIS_ENABLE_GRPC_V2 "Enable version 2 of the GRPC API in server" ON)
```

Comment:
Sorry, I misunderstood your question. Leave this OFF here but change Dockerfile to set it to on for our build.